### ISSUE TITLE
Visuals have been drilled down into 2019

### ISSUE THREAD

**Issue by mattantaliss** [mattfeks/Snowflake-PowerBI#1]
I was using the sample .pbix file as a starting point for our actual Snowflake data, and for a long time I could not get past a number of extra filters being applied to the visuals. There were three separate 'Year == 2019' filters being applied. I managed to stumble across a Q&A that mentioned the need to Drill Up on some visuals to get rid of filters that would not allow the user to remove them. I then tried doing that Drill Up on each of the visuals in the modified version of the sample .pbix file and that managed to resolve everything for me.

In short, the sample .pbix file has some visuals that have been drilled down into 2019, and that breaks things when using it as a starting point and pulling in 2020 data. Each of the visuals in the sample .pbix should be drilled up to remove the filtering that locks results into 2019.
